__tests__/test.js

```diff
@@ -132,43 +132,129 @@ describe("/api/articles", () => {
+      return request(app)
+        .get("/api/articles/10000086/comments")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid article ID");
+        });
+    });
+    test("should respond with 400 and a msg of invalid article_id if passed bad request", () => {
+      return request(app)
+        .get("/api/articles/78787HELLO/comments")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("article_id is not a number");
+        });
+    });
   });
-  test("GET request should respond with 404 and a msg of invalid article_id if passed bad request", () => {
-    return request(app)
-      .get("/api/articles/78787HELLO/comments")
-      .expect(400)
-      .then(({ body }) => {
-        expect(body.msg).toBe("article_id is not a number");
+  describe("POST request", () => {
+    test("should respond with 201, accepting an object with username and body properties and respond with the comment object with the following properties: comment_id, votes, created_at, author, body, article_id", () => {
+      const newComment = {
+        username: "icellusedkars",
+        body: "W article",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          const comment = body;
+          expect(comment.author).toBe("icellusedkars");
+          expect(comment.comment_id).toBe(19);
+          expect(comment.votes).toBe(0);
+          expect(comment).toHaveProperty("created_at");
+          expect(comment.body).toBe("W article");
+          expect(comment.article_id).toBe(2);
+        });
+    });
+    test("should respond with 404 and a msg of invalid article_id  article_id is not found", () => {
+      const newComment = {
+        username: "icellusedkars",
+        body: "W article",
+      };
+      return request(app)
+        .post("/api/articles/10000086/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("article ID not found");
+        });
+    });
+    test("should return with 400 and a msg of article ID is not a number", () => {
+      const newComment = {
+        username: "icellusedkars",
+        body: "W article",
+      };
+      return request(app)
+        .post("/api/articles/100dwain00086/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid article ID");
+        });
+    });
+    test("should return with 400 and a msg of invalid comment object if the object does not have a username property", () => {
+      const newComment = {
+        username: "icellusedkars"
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid comment object");
+        })
+    });
```

Comment:
You can combine these two tests for a Status 400 - Bad request: Request body incomplete/missing required field(s) of username or body.